README.md

```diff
@@ -179,3 +179,16 @@ This recipe downloads openssl source package using _apt://_. This recipe shows h
 This recipe downloads openssl source package using _http://_. This recipe shows how to change build option and how to add new patch for source code.
 - openssl-git  
 This recipe downloads openssl source package using _git://_. This recipe shows how to change build option.
+
+# License
+
+License of meta-emlinux is MIT licensed unless otherwise stated.
+Source code included in tree for individual recipes is under the LICENSE stated in the associated recipe (.bb file) unless otherwise stated.
```

Comment:
```
All metadata is MIT licensed unless otherwise stated. Source code
included in tree for individual recipes is under the LICENSE stated in
the associated recipe (.bb file) unless otherwise stated.

```

Reply:
OK. I will replace `s/License of meta-emlinux/All metadata/`.

Reply:
I've force-pushed its replacement.